app/models/volunteer.rb

```diff
@@ -2,6 +2,10 @@
 
 class Volunteer < ApplicationRecord
   belongs_to :user
+<<<<<<< HEAD
```

Comment:
Again, this file is screwed up because the git version conflicts have not been resolved.  It can't run as it is.